automated install/basic-install.sh

```diff
@@ -2008,8 +2169,12 @@ installPihole() {
         fi
     fi
     # For updates and unattended install.
-    if [[ "${useUpdateVars}" == true ]]; then
-        accountForRefactor
+    if [[ "${runUnattended}" == true ]]; then
+        if [[ -f "${setupVars}" ]]; then
+            chmod 644 "${setupVars}"
+            accountForRefactor
```

Comment:
note for myself more than anything else. : this function can probably go away, now. I don't even remember the version that it was introduced in, but I very very much doubt that anyone is still updating from that version!

Reply:
It was a good spot to call the validate_setupVars() function.